test/ui-test/src/screens/StatusChatScreen.py

```diff
@@ -54,6 +57,12 @@ def send_message(self, message: str):
         type(ChatComponents.MESSAGE_INPUT.value, message)
         press_enter(ChatComponents.MESSAGE_INPUT.value)
         
+    def type_message(self, message: str):
```

Comment:
It clarifies the name, to know where we are typing the name.
```suggestion
    def type_message_in_chat_input(self, message: str):
```